router/batchrouter/asyncdestinationmanager/snowpipestreaming/channel.go

```diff
@@ -125,6 +126,9 @@ func (m *Manager) createChannel(
 		return resp, nil
 	case internalapi.ErrValidationError, internalapi.ErrAuthenticationFailed, internalapi.ErrRoleDoesNotExistOrNotAuthorized, internalapi.ErrDatabaseDoesNotExistOrNotAuthorized:
 		return nil, fmt.Errorf("%w, %w", errAuthz, err)
+	case internalapi.ErrUnknownError:
+		// This case was added to handle the case where streaming does not support certain column types (e.g., AUTOINCREMENT)
+		return nil, fmt.Errorf("%w: creating channel with code %s, message: %s and error: %s", errAbort, resp.Code, resp.SnowflakeAPIMessage, resp.Error)
```

Comment:
Should we be more specific error? Like status code or error format? https://knowledge.informatica.com/s/article/000233095?language=en_US
Aborting things in case of unknown errors, doesn't seems right.

Reply:
Updated the check to abort only in case of a specific API code